mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -34,7 +34,7 @@ var stopCmd = &cobra.Command{
 	Long:  `Stop all Meshery containers, remove their instances and prune their connected volumes.`,
 	Args:  cobra.NoArgs,
 	PreRunE: func(cmd *cobra.Command, args []string) error {
-		return utils.PreReqCheck()
+		return utils.PreReqCheck("stop")
```

Comment:
I was just wondering will it be good to use `stopCmd.Use` to pass as a parameter instead of `"stop"` string to avoid human error and at all the other places you have paased a separate string instead of `anyCmd.Use`?

Reply:
Sure @NupurThakur27, we can do that, as long as anyCmd.Use is a string, I was not sure of this convention

Reply:
@NupurThakur27 Done, please check!